test/gtest/common/test_helpers.h

```diff
@@ -31,6 +25,124 @@ extern "C" {
 #include <stdint.h>
 
 
+#ifndef UINT16_MAX
+#define UINT16_MAX (65535)
```

Comment:
why is this needed ? can use `#define __STDC_LIMIT_MACROS` before including `<stdint.h>`

Reply:
`#define __STDC_LIMIT_MACROS` does not help, the problem is that we include internal C-headers directly from gtests. without this definition compilation is broken on ARM, If it's critical, i would prefer to play with this out of scope of this PR.

Reply:
what is the problem to add this definition? we already do it in multiple tests (e.g ucm/malloc_hook.cc)

Reply:
Adding in this file doesn't help, i did not try to do that on every compilation failure in every test.

Reply:
BTW, doesn't seem like UINT16_MAX is even used anywhere in gtest..
also, AFAIK the C++ way is using `std::numeric_limits<uint16_t>::max()`

Reply:
Yes, but it's used in internal (non API) UC<X> headers included directly from gtests.

Reply:
Probable right way is to include <stdint.h> in these library files, not in gtests

Reply:
IC. so need to either define __STDC_LIMIT_MACROS in all tests which include internal UCX headers and use these macros, or add it to gtest compilation command line CPPFLAGS (-D__STDC_LIMIT_MACROS)

Reply:
Let's do it out of this PR. Really, this code was not even changed.

Reply:
ok